README.md

```diff
@@ -256,14 +256,14 @@ This is how fields in the Page Spec are populated:
 
 | Page Spec field           | Source field in Fingerprint Pro `result` or metadata passed to `segment`                                                                 |
 | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
-| `anonymousId`             | [`result.visitorId`](https://dev.fingerprint.com/docs/js-agent#visitorid) - Fingerprint `visitorId`                                      |
+| `anonymousId`             | [`result.visitorId`](https://dev.fingerprint.com/reference/get-function#visitorid) - Fingerprint `visitorId`                                      |
 | `userId`                  | `identify.userId` - your internal user ID                                                                                                |
-| `context.ip`              | [`result.ip`](https://dev.fingerprint.com/docs/js-agent#ip) - IP address                                                                 |
-| `context.browserDetails`  | [`result.browserDetails`](https://dev.fingerprint.com/docs/webhooks#identification-webhook-object-format) - Browser, OS name and version |
-| `context.incognito`       | [`result.incognito`](https://dev.fingerprint.com/docs/js-agent#incognito) - Incognito mode                                               |
-| `context.confidenceScore` | [`result.confidenceScore`](https://dev.fingerprint.com/docs/js-agent#confidence) - Confidence score                                      |
-| `context.requestId`       | [`result.requestId`](https://dev.fingerprint.com/docs/js-agent#requestid) - ID of Fingerprint request                                    |
-| `properties.url`          | [`result.url`](https://dev.fingerprint.com/docs/webhooks#identification-webhook-object-format) - page URL                                |
+| `context.ip`              | [`result.ip`](https://dev.fingerprint.com/reference/get-function#ip) - IP address                                                                 |
+| `context.browserDetails`  | [`result.browserDetails`](https://dev.fingerprint.com/docs/webhooks#json-example) - Browser, OS name and version |
+| `context.incognito`       | [`result.incognito`](https://dev.fingerprint.com/reference/get-function#incognito) - Incognito mode                                               |
+| `context.confidenceScore` | [`result.confidenceScore`](https://dev.fingerprint.com/reference/get-function#confidence) - Confidence score                                      |
```

Comment:
Is the text inside documentation for confidence, updated with our new Confidence score v1.1?

Reply:
Yes!